package.json

```diff
@@ -14,13 +14,23 @@
     "@fortawesome/free-brands-svg-icons": "^5.13.0",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/vue-fontawesome": "^0.1.9",
+    "@popperjs/core": "^2.3.3",
```

Comment:
Is this used?

Reply:
Not anymore. I used it before only to find it not working the way I wanted. This can be deleted

Reply:
Could you delete it?

Reply:
okay

Reply:
I deleted the code in my branch already. Do i have to make a new pull request?

Reply:
No.